dev/reference/best-practices/haproxy.md

```diff
@@ -0,0 +1,222 @@
+   timeout connect  2s                     # The maximum time to wait for a connection attempt to a backend server to succeed. It should be set to a shorter time if the server is located on the same LAN as HAProxy.
+   timeout client 30000s                   # The maximum inactivity time on the client side.
+   timeout server 30000s                   # The maximum inactivity time on the server side.
+
+listen admin_stats                         # The name of the Stats page reporting information from frontend and backend. You can customize the name according to your needs.
+   bind 0.0.0.0:8080                       # The listening port.
+   mode http                               # The monitoring mode.
+   option httplog                          # Enables HTTP logging.
+   maxconn 10                              # The maximum number of concurrent connections.
+   stats refresh 30s                       # Automatically refreshes the Stats page every 30 seconds.
+   stats uri /haproxy                      # The URL of the Stats page.
+   stats realm HAProxy                     # The authentication realm of the Stats page.
+   stats auth admin:pingcap123             # User name and password in the Stats page. You can have multiple user names.
+   stats hide-version                      # Hides the version information of HAProxy on the Stats page.
+   stats admin if TRUE                     # Manually enables or disables the backend server. <span class="version-mark">New in HAProxy 1.4.9 or later versions</span>
+
+listen tidb-cluster                        # Database load balancing.
+   bind 0.0.0.0:3390                       # Floating IP addresses and listening ports.
+   mode tcp                                # HAProxy uses layer 4, the transport layer.
+   balance leastconn                       # The server with the smallest number of connections receives the connection. `leastconn' is recommended where long sessions are expected, such as LDAP, SQL and TSE, rather than protocols using short sessions, such as HTTP. The algorithm is dynamic, which means that server weights might be adjusted on the fly for slow starts for instance.
+   server tidb-1 10.9.18.229:4000 check inter 2000 rise 2 fall 3       # Detects port 4000 at a frequency of once every 2000 milliseconds. If it is detected as successful twice, the server is considered available; if it is detected as failed three times, the server is considered unavailable.
+   server tidb-2 10.9.39.208:4000 check inter 2000 rise 2 fall 3
+   server tidb-3 10.9.64.166:4000 check inter 2000 rise 2 fall 3
+```
+
+### Use `systemd` to start HAProxy
+
+There are two methods to start HAProxy.
+
+Method 1: Execute `haproxy`.
+
+{{< copyable "shell-regular" >}}
+
+```bash
+haproxy -f /etc/haproxy/haproxy.cfg
+```
+
+Method 2: Use `systemd` to start HAProxy. `/etc/haproxy/haproxy.cfg` is read by default (recommended).
```

Comment:
It seems that this method 2 is what the heading says. Please check whether the method 1 is a sub-step of method 2, because it's more like one step.